frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -321,6 +325,40 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(trackerRemoval);
   }
 
+  const phoneAnnouncement: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-phone-header"),
+    snippet: l10n.getString("whatsnew-feature-phone-snippet"),
+    content: (
+      <WhatsNewContent
+        description={l10n.getString("whatsnew-feature-phone-description")}
+        heading={l10n.getString("whatsnew-feature-phone-header")}
+        image={PhoneMaskingHero.src}
+        cta={{
+          type: "upgrade",
+          content: l10n.getString("whatsnew-feature-phone-upgrade-cta"),
+          onClick: () => trackPurchaseStart(),
+          gaViewPing: {
+            category: "Purchase Button",
+            label: "whatsnew-phone-promo",
+          },
+        }}
+      />
+    ),
+    hero: PhoneMaskingHero.src,
+    icon: PhoneMaskingIcon.src,
+    dismissal: useLocalDismissal(`whatsnew-feature_phone_${props.profile.id}`),
+    announcementDate: {
+      year: 2022,
+      month: 10,
+      day: 11,
+    },
+  };
+
+  // Only show its announcement if phone masking is live:
+  if (isFlagActive(props.runtimeData, "phones")) {
```

Comment:
I think we'll also probably only want to show this if the user hasn't already purchased the phones plan?

```suggestion
  if (isFlagActive(props.runtimeData, "phones") && !props.profile.has_phone) {
```

(Not sure if this is the correct check for that, but I presume you know which is if it's not :) )

Reply:
Probably related, because I'm using a Premium account: now if I do see the News announcement, there's no CTA:

![image](https://user-images.githubusercontent.com/4251/186651570-945128d3-5c75-43f8-a9e2-8d0c7c2d79d6.png)

Though seeing that now, I guess maybe it's intentional to show it anyway? Doesn't look too weird.